### DISCUSSION TITLE
Make an image with Labor beta fritzos

### DISCUSSION THREAD

**Discussion by ektorasdj** [Freetz-NG/freetz-ng#356] [Allgemein] [answered]
Hello and thank you for this fine tool.
I have been using it for months but now i want to modify
the new labor version 7.27 89860 for fritzbox 7530.
Where i can change the setting to choose the beta files or 
my local image of the labor firtzos?
Thanks and sorry if this has been asked before,
I dont know german and search is hard that way.

**Comment by ektorasdj** [comment, marked as answer]
Found IT!!!

first you choose level of user competence : expert.

Then download options-> Override firmware source - > and then you change the firmware name in the last option that says firmware source.
The labor firmware should be on your freetz-ng folder ->dl->fw

**Comment by fda77** [comment]
As long as nobody adds the new labor fws selectable by menuconfig, this is the best way
